.github/workflows/ci.yaml

```diff
@@ -58,16 +59,24 @@ jobs:
         uses: actions/cache@v2
```

Comment:
note that this block, L59 to L79, could be a single simple action with `ramsey/install` as shown [here](https://github.com/infection/infection/pull/1781/files#diff-87ab0e37e52f2d5a98de1c719e9222197e1f875c57ed290a6b10f0c640e5b001R33).